packages/react-mutation-mapper/src/util/OncoKbUtils.ts

```diff
@@ -345,6 +345,19 @@ export function getTumorTypeName(tumorType?: TumorType) {
     }
 }
 
+export function getTumorTypeNameWithExclusionInfo(
+    tumorType?: TumorType,
+    excludedTumorTypes?: TumorType[]
+) {
+    let name = getTumorTypeName(tumorType);
+    if ((excludedTumorTypes || []).length > 0) {
+        name += ` (excluding ${excludedTumorTypes!
```

Comment:
string concat makes me uncomfortable :) how about this?

```suggestion
        name = `${name} (excluding ${excludedTumorTypes!
```